file_paths.md

```diff
@@ -197,13 +210,25 @@ user or group.
             puppet.conf *                     # :config
             routes.yaml                       # :route_file
             ssl                               # :ssldir
-    -------------------------------------------------------------------------------------------------------               
+    -------------------------------------------------------------------------------------------------------
         var *
             log                               # :logdir                      same
                 puppet.log                    # not enabled by default
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.conf                    # pxp-agent configuration file
+            modules *                         # stores configuration files for pxp-agent modules
```

Comment:
I think we have enough buy-in on the email thread for this to be a part of packaging, so my prior comment here is invalidated.

Now on to determining if we're adding a submodule to `pxp-agent` or collapsing repos.